CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changes
 
+## 1.8.X
```

Comment:
Since we haven't released `1.8.3`, you can just merge the message from `1.8.3` into `1.8.X`.

Reply:
Done